native/src/main/java/io/ballerina/stdlib/graphql/runtime/utils/Utils.java

```diff
@@ -66,6 +67,9 @@ public static BError createError(String message, String errorTypeName, BError ca
     }
 
     public static boolean isContext(Type type) {
+        if (type.getTag() == TypeTags.ARRAY_TAG) {
+            return false;
+        }
```

Comment:
Why do we have to specifically check for the array types here? Is this check needed?

Reply:
When we get an array type input like `float[][]`, `type.Package` value is not present. Due to that following if condition will cause to a NPE. `if (type.getPackage().getOrg() == null || type.getPackage().getName() == null) {`.

Reply:
Then let's add a check like this:

```java
if (type.getPackage() == null) {
    return false;
}

if (type.getPackage().getOrg() == null || type.getPackage().getName() == null) {
    return false;
}
```